doc/seg-schema.json

```diff
@@ -1,152 +1,197 @@
   "@context": "http://qiicr.org/dcmqi/contexts/dcmqi.jsonld",
-  "id" : "http://qiicr.org/schema/dcmqi/seg-schema.json",
+  "id": "https://raw.githubusercontent.com/che85/dcmqi/fixSchema/doc/seg-schema.json#",
   "$schema": "http://json-schema.org/draft-04/schema#",
-
-  "seriesAttributes": {
-    "type" : "object",
-    "properties" : {
-
-      "ContentCreatorName":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/PN"},
-        {"properties" : {"default" : "Reader1"}}]
-      },
-
-      "ClinicalTrialSeriesID":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
-        {"properties" : {"default" : "Session1"}}]
-      },
-
-      "ContentLabel":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/CS"},
-        {"properties" : {"default" : "SEGMENTATION"}}]
-      },
-
-      "ContentDescription":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
-        {"properties" : {"default" : "Image segmentation"}}]
-      },
-
-      "ClinicalTrialCoordinatingCenterName":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
-        {"properties" : {"default" : "dcmqi"}}]
-      },
-
-      "ClinicalTrialTimePointID":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
-        {"properties" : {"default" : "1"}}]
-      },
-
-      "SeriesDescription":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/LO"},
-        {"properties" : {"default" : "Segmentation"}}]
-      },
-
-      "SeriesNumber":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/IS"},
-        {"properties" : {"default" : "300"}}]
-      },
-
-      "InstanceNumber":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/IS"},
-        {"properties" : {"default" : "1"}}]
-      },
-
-      "BodyPartExamined":
-      { "allOf" :
-        [{"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/CS"},
-        {"properties" : {"default" : ""}}]
-      }
-    }
-  },
-
-  "segmentAttributes": {
-    "type" : "array",
-    "items" : {
-      "type" : "object",
-      "properties" : {
-        "LabelID": { "type" : "number", "default" : 1},
-        "SegmentDescription": { "type" : "string", "required" : false},
-
-        "SegmentedPropertyCategoryCodeSequence": {
-          "allOf" : [
-            {"$ref" : "http://qiicr.org/schema/dcmqi/common-schema.json#/definitions/codeSequence"},
-            {"properties" :
-             {"CodeValue" : {"default": "T-D0050"},
-              "CodingSchemeDesignator" : {"default": "SRT"},
-              "CodeMeaning" : {"default": "Tissue"}}
+  "properties": {
```

Comment:
oh, I see, I missed it - I thought it is present only in this schema, but it also shows up for pm-schema. Alright